app/controllers/users_controller.rb

```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+    render 'index'
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @recent_posts = @user.recent_posts
+    render 'show'
+  end
```

Comment:
- [ ] Great job on adding the `index` and `show` actions. However, there seems to be an issue currently, as trying to access the show page results in an error, as illustrated below 👇

_Hint: You can leave all the actions empty for now_

![image](https://github.com/forLoop94/blogApp/assets/106119105/63fc61ec-7973-4ba3-89ba-a8af904c881a)